app/views/quizzes/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with(model: quiz, local: true) do |form| %>
+  <% if quiz.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(quiz.errors.count, "error") %> prohibited this quiz from being saved:</h2>
+
+      <ul>
+        <% quiz.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= form.hidden_field :grammar_id %>
+
+  <div class="field">
+    <%= form.label :tokyo %>
+    <%= form.text_field :tokyo %>
```

Comment:
Could you give these a placeholder so we can see an example when we are creating these?

Reply:
👀 👂 

<img width="594" alt="Screen Shot 2021-01-27 at 4 28 22 PM" src="https://user-images.githubusercontent.com/60206746/106072243-aeefc780-60bc-11eb-9648-26a0979b9f1c.png">